src/network_generator.py

```
@@ -85,7 +84,7 @@ def generate_network(
             return True
 
         except Exception as e:
-            logger.error(f"Error occurred while generating network: {e}", exc_info=True)
+            logger.exception(f"Error occurred while generating network: {e}")
```

Ruff (0.14.2)

Redundant exception object included in `logging.exception` call

(TRY401)